sdk/core/azure-core/azure/core/utils/_utils.py

```diff
@@ -161,3 +162,25 @@ def __eq__(self, other: Any) -> bool:
 
     def __repr__(self) -> str:
         return str(dict(self.items()))
+
+
+def parse_status_link(status_link: str, lro_options: Optional[Dict[str, Any]] = None) -> Tuple[str, Dict[str, Any]]:
```

Comment:
Note that I didn't say "utils", I said "SansIOBasePolling" (or at least the same file). I don't want this code to be too far from it's usage, that creates import complexity unnecessarily.

Reply:
Got it. Compared with adding statistic method for "SansIOBasePolling", it is better to only put the method in same file.